client/src/partials/debtor_groups/debtor_groups.html

```diff
@@ -0,0 +1,281 @@
+                      ng-false-value="0"
+                      ng-model="DebtorGroupCtrl.debtorGroup.locked">
+                    {{ "DEBTOR_GRP.LOCKED" | translate }}
+                  </label>
+                </div>
+
+                <div class="form-group col-xs-6">
+                  <label class="control-label">
+                    <input
+                      type="checkbox"
+                      name="apply_discounts"
+                      ng-true-value="1"
+                      ng-false-value="0"
+                      ng-model="DebtorGroupCtrl.debtorGroup.apply_discounts">
+                    {{ "DEBTOR_GRP.APPLY_DISCOUNTS" | translate }}
+                  </label>
+                </div>
+
+                <div class="form-group col-xs-6">
+                  <label class="control-label">
+                    <input
+                      type="checkbox"
+                      name="apply_billing_services"
+                      ng-true-value="1"
+                      ng-false-value="0"
+                      ng-model="DebtorGroupCtrl.debtorGroup.apply_billing_services">
+                    {{ "DEBTOR_GRP.APPLY_BILLING_SERVICES" | translate }}
+                  </label>
+                </div>
+
+                <div class="form-group col-xs-6">
+                  <label class="control-label">
+                    <input
+                      type="checkbox"
+                      name="apply_subsidies"
+                      ng-true-value="1"
+                      ng-false-value="0"
+                      ng-model="DebtorGroupCtrl.debtorGroup.apply_subsidies">
+                    {{ "DEBTOR_GRP.APPLY_SUBSIDIES" | translate }}
+                  </label>
+                </div>
+
+              </fieldset>
+
+              <!-- contact info -->
+              <fieldset>
+                <legend>{{ "COLUMNS.CONTACT" | translate }}</legend>
+                <!-- phone  -->
+                <div class="form-group">
+                  <label class="control-label">
+                    {{ "COLUMNS.PHONE" | translate }}
+                  </label>
+                  <input
+                    class="form-control"
+                    name="phone"
+                    placeholder="{{ 'COLUMNS.PHONE' | translate }}..."
+                    ng-model="DebtorGroupCtrl.debtorGroup.phone">
+                </div>
+
+                <!-- email  -->
+                <div class="form-group">
+                  <label class="control-label">
+                    {{ "COLUMNS.EMAIL" | translate }}
+                  </label>
+                  <input
+                    class="form-control"
+                    name="email"
+                    placeholder="{{ 'COLUMNS.EMAIL' | translate }}..."
+                    ng-model="DebtorGroupCtrl.debtorGroup.email">
+                </div>
+
+                <!-- debtor group location  -->
+                <bh-location-select
+                  id="origin-location-id"
```

Comment:
In the future, you do not need to put an id on this element, if it is the only `<bh-location-select>` on the page.